### ISSUE TITLE
Unclassified error occured while trying to activate license for personal license

### ISSUE THREAD

**Issue by ayutaz** [game-ci/unity-actions#101]
**Bug description**

When I build for Android using unity-builder on Github actions, I get an error message "Unclassified error occured while trying to activate license.

The error message displayed is as follows.

```
Successfully built dfe6661bcdca
Successfully tagged unity-builder:2020.3.5f1-android-0
/usr/bin/docker run --workdir /github/workspace --rm --env UNITY_LICENSE --env UNITY_LICENSE_FILE --env UNITY_EMAIL --env UNITY_PASSWORD --env UNITY_SERIAL --env UNITY_VERSION=2020.3.5f1 --env PROJECT_PATH=. --env BUILD_TARGET=Android --env BUILD_NAME=Android --env BUILD_PATH=build/Android --env BUILD_FILE=Android.apk --env BUILD_METHOD= --env VERSION=0.9.26 --env ANDROID_VERSION_CODE=9026 --env ANDROID_KEYSTORE_NAME= --env ANDROID_KEYSTORE_BASE64= --env ANDROID_KEYSTORE_PASS= --env ANDROID_KEYALIAS_NAME= --env ANDROID_KEYALIAS_PASS= --env CUSTOM_PARAMETERS= --env HOME=/github/home --env GITHUB_REF --env GITHUB_SHA --env GITHUB_REPOSITORY --env GITHUB_ACTOR --env GITHUB_WORKFLOW --env GITHUB_HEAD_REF --env GITHUB_BASE_REF --env GITHUB_EVENT_NAME --env GITHUB_WORKSPACE=/github/workspace --env GITHUB_ACTION --env GITHUB_EVENT_PATH --env RUNNER_OS --env RUNNER_TOOL_CACHE --env RUNNER_TEMP --env RUNNER_WORKSPACE --volume /var/run/docker.sock:/var/run/docker.sock --volume /home/runner/work/_temp/_github_home:/github/home --volume /home/runner/work/_temp/_github_workflow:/github/workflow --volume /home/runner/work/HeartDrivenCasualGame/HeartDrivenCasualGame:/github/workspace unity-builder:2020.3.5f1-android-0
Requesting activation (personal license)
Unclassified error occured while trying to activate license.
Exit code was: 1
Error: The process '/usr/bin/docker' failed with exit code 1
```

I also referred to the following issue, but it did not solve the problem.
https://github.com/game-ci/unity-builder/issues/199
https://github.com/game-ci/unity-builder/issues/204
#70 

**How to reproduce**

The configuration file for github actions is shown below.

``` 

name: Acquire activation file
on:
  workflow_dispatch: {}
jobs:
  activation:
    name: Request manual activation file 🔑
    runs-on: ubuntu-latest
    steps:
      # Request manual activation file
      - name: Request manual activation file
        id: getManualLicenseFile
        uses: game-ci/unity-request-activation-file@v2.0-alpha-1
        with:
          unityVersion: 2020.3.5f1
      # Upload artifact (Unity_v20XX.X.XXXX.alf)
      - name: Expose as artifact
        uses: actions/upload-artifact@v2
        with:
          name: ${{ steps.getManualLicenseFile.outputs.filePath }}
          path: ${{ steps.getManualLicenseFile.outputs.filePath }}

```

```
# This is a basic workflow to help you get started with Actions

name: Test and Build,Release APK

env:
  UNITY_LICENSE: ${{ secrets.UNITY_LICENSE }}

# Controls when the action will run. 
on: [push]
  # Triggers the workflow on push or pull request events but only for the main branch

# A workflow run is made up of one or more jobs that can run sequentially or in parallel
jobs:
  build:
    runs-on: ubuntu-latest

    steps:
    - name: Checkout
      uses: actions/checkout@v2
      with:
        lfs: false
        clean: false
        
    # Cache
    - uses: actions/cache@v2
      with:
        path: Library
        key: Library

    # Test
#     - name: Run tests
#       uses: game-ci/unity-test-runner@v2
#       env:
#         UNITY_LICENSE: ${{ secrets.UNITY_LICENSE }}
#       with:
#         githubToken: ${{ secrets.GITHUB_TOKEN }}

    # Build
    - name: Build project
      uses: game-ci/unity-builder@v2.0-alpha-6
      with:
        unityVersion: 2020.3.5f1
        targetPlatform: Android

    # Output
    - uses: actions/upload-artifact@v2
      with:
        name: Build
        path: build
```

For other files, please refer to the following repositories
https://github.com/ayutaz/HeartDrivenCasualGame

**Expected behavior**

Build successfully.

**Additional details**

* Unity 2020.3.5f1
* Build for Android

thank you

**Comment by MohdAmmarNaeem** [comment]
@webbertakken I am still getting same activation error as @ayutaz stated. What can I do here, any solutions?

**Comment by webbertakken** [comment]
> @webbertakken I am still getting same activation error as @ayutaz stated. What can I do here, any solutions?

We generally haven't seen people with this error anymore since it was fixed a while ago, meaning that mostly this now works.

You can do two things:
- Make sure your workflows are completely up-to-date
- Narrow it down to a specific cause (editor version, different projects, and so on). Try to be as specific as possible to make your bug report actionable.

If it is a specific Unity version that's causing this, you can report it to Unity and [our docker repo](https://github.com/game-ci/docker/issues).

If it is specific to Unity Actions but not the docker images, you can report a new issue in this repository, referencing this one.

**Comment by GabLeRoux** [comment]
In case someone reaches this page from a Google search for "Unclassified error occured while trying to activate license", go here:
https://github.com/game-ci/unity-builder/issues/485#issuecomment-1749412929

👍
